tests/client/util.rs

```diff
@@ -116,16 +131,16 @@ async fn init_oidc() -> (String, Sender<()>, JoinHandle<()>) {
                                 let auth = req
                                     .header("Authorization")
                                     .expect("Authorization header missing");
-                                assert_eq!(
-                                    auth.as_str().split_once(' '),
-                                    Some(("Bearer", "test-token")),
-                                );
-                                json_response(&CoreUserInfoClaims::new(
-                                    StandardClaims::new(SubjectIdentifier::new(
-                                        "test|subject".into(),
-                                    )),
-                                    EmptyAdditionalClaims {},
-                                ))
+                                if auth.as_str().split_once(' ') != Some(("Bearer", "test-token")) {
+                                    Ok(Response::new(StatusCode::Unauthorized))
+                                } else {
+                                    json_response(&CoreUserInfoClaims::new(
+                                        StandardClaims::new(SubjectIdentifier::new(
+                                            "test|subject".into(),
+                                        )),
+                                        EmptyAdditionalClaims {},
+                                    ))
+                                }
```

Comment:
this chunk does not match the PR or patch description

Reply:
I mentioned it in passing in the description, I've updated it to be more thorough.